dataverse/orgsvc/C#-NETCore/ElasticTables/ElasticTableSamples/ElasticTableSamples/Utility.cs

```diff
@@ -0,0 +1,248 @@
+﻿using Microsoft.PowerPlatform.Dataverse.Client;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Messages;
+using Microsoft.Xrm.Sdk.Metadata;
+using Newtonsoft.Json.Linq;
+
+namespace PowerPlatform.Dataverse.CodeSamples
+{
+    internal class Utility
+    {
+        private static string SensorDataSchemaName = "contoso_SensorData";
+        private static string DeviceIdSchemaName = "contoso_DeviceId";
+        private static string SensorTypeSchemaName = "contoso_SensorType";
+        private static string ValueSchemaName = "contoso_Value";
+        private static string TimeStampSchemaName = "contoso_TimeStamp";
+        private static string EnergyConsumptionSchemaName = "contoso_EnergyConsumption";
+        private static string SensorDataLogicalName = SensorDataSchemaName.ToLower();
+
+        internal static void CreateSensorDataEntity(ServiceClient client)
+        {
+            Console.WriteLine($"Creating {SensorDataSchemaName} entity.. \n");
+            EntityMetadata entityMetadata = new EntityMetadata
```

Comment:
You don't seem to be using this `entityMetadata` anywhere.
I know we want to use it in the future.
Can you remove or add comment about why it is here?

Reply:
yes, my bad, forgot to remove it.